src/webgpu/api/validation/capability_checks/features/texture_component_swizzle_operation.spec.ts

```diff
@@ -0,0 +1,675 @@
+export const description = `
+Operational tests for the 'texture-component-swizzle' feature.
```

Comment:
Should it live in `api/operation` instead of `api/validation`?

Reply:
yea, maybe. Preference for location?

* `src/webgpu/api/operation/features/texture_component_swizzle.spec.ts` ?
* `src/webgpu/api/operation/texture/texture_component_swizzle.spec.ts` ?
* other?